run_ci.sh

```diff
@@ -38,6 +38,9 @@ sudo sed -i -e '/keepcache=0/d' /etc/yum.conf
 sudo mkdir -p /opt/data/yumcache
 sudo mount -o bind /opt/data/yumcache /var/cache/yum
 
+# TEST no merge
+git clone https://github.com/knowncitizen/facet -b openshift-metalkube-migration
```

Comment:
this can be updated as https://github.com/openshift-metalkube/facet/pull/49 has been merged